lib/LedgerSMB/Template.pm

```diff
@@ -568,6 +568,7 @@ sub _render {
     $cleanvars->{UNESCAPE} = sub { return $unescape->(@_); }
         if ($unescape && !$self->{no_escape});
     $cleanvars->{text} = sub { return $self->_maketext($escape, @_); };
+    $cleanvars->{decorated_text} = sub { return $self->{locale}->maketext(@_); };
```

Comment:
You're introducing a function `decorated_text()` which is a misnomer (it doesn't do anything about decoration: it translates), but it's also not necessary (see my comment above).